kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -58,6 +68,45 @@
 log = logging.getLogger('collector')
 
 
+class ExpiredPodFilter(object):
+    """
+    Allows to filter old pods out of the podlist by providing a decoding hook
+    """
+    def __init__(self, cutoff_date):
+        self.expired_count = 0
+        self.cutoff_date = cutoff_date
+
+    def json_hook(self, pod):
```

Comment:
since pod is not always a `pod`, maybe it is better to rename the arg: resource, obj... WDYT?